draft-ietf-acme-acme.md

```diff
@@ -2761,6 +2761,17 @@ server is in a default virtual host configuration.  Conversely, if the TLS
 server returns an unrecognized_name alert, then this is an indication that the
 server is not in a default virtual host configuration.
 
+## Token Entropy
+
+The http-01, tls-sni-02 and dns-01 validation methods mandate the usage of
+a random token value to uniquely identify the challenge. The value of the token
+is required to contain at least 182 bits of entropy for the following security
```

Comment:
Thanks for catching that :mag: :keyboard:. Fixed in https://github.com/ietf-wg-acme/acme/pull/284/commits/39d777e11e95841c569acd8465f3b1903f83f0b7